apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -101,38 +149,24 @@ class Comment extends Component {
             <span style={styles.timestamp}>
               {this.renderFormattedTimestamp(timestampString)}
             </span>
-            {(viewAsTeacher || !viewAsCodeReviewer) && (
-              <i
-                className="fa fa-ellipsis-h"
-                style={styles.ellipsisMenu}
-                onClick={() =>
-                  this.setState({
-                    isShowingCommentOptions: !isShowingCommentOptions
-                  })
-                }
-              >
-                {isShowingCommentOptions && (
-                  <CommentOptions
-                    isResolved={isResolved}
-                    onResolveStateToggle={() => this.onResolve()}
-                    onDelete={() => this.onDelete()}
-                  />
-                )}
-              </i>
-            )}
+            <InlineDropdownMenu icon="fa fa-ellipsis-h">
+              {this.getListItems()}
+            </InlineDropdownMenu>
           </span>
         </div>
-        <div
-          className={'code-review-comment-body'}
-          style={{
-            ...styles.comment,
-            ...(isFromTeacher && styles.commentFromTeacher),
-            ...((isFromOlderVersionOfProject || isResolved) &&
-              styles.lessVisibleBackgroundColor)
-          }}
-        >
-          {commentText}
-        </div>
+        {!(isResolved && hideWhileResolved) && (
+          <div
+            className={'code-review-comment-body'}
```

Comment:
nit: brackets are unnecessary because this isn't a javascript expression -- can use "" instead

Reply:
Thanks for catching that!